spec.html

```diff
@@ -11892,7 +11892,7 @@
           1. Let _tailCall_ be IsInTailPosition(_thisCall_).
           1. Return ? EvaluateDirectCall(_func_, _thisValue_, |Arguments|, _tailCall_).
         </emu-alg>
-        <p>A |CallExpression| evaluation that executes step 3.a.vi is a <dfn>direct eval</dfn>.</p>
+        <p>A |CallExpression| evaluation that executes step 3.a.vii is a <dfn>direct eval</dfn>.</p>
```

Comment:
Referring to a step seems like asking for trouble unless you're linking to a snapped version. I too wish there were a way to link to steps in a stable manner without requiring each step to be annotated with a unique id. You could consider a hash of the step contents but then minor editorial changes would cause problems...